spec/index.bs

```diff
@@ -2088,8 +2101,46 @@ enum GPUTextureAspect {
         </pre>
 
         **Returns:** |view|, of type {{GPUTextureView}}.
-
-        Issue: write definition. |this| |descriptor| |view|
+        1. [=resolve GPUTextureViewDescriptor defaults=]
+        1. If any of the following conditions are unsatisfied, generate a validation
+            error and return an [=invalid=] {{GPUTextureView}}.
+            <div class=validusage>
+                - 0 &le; {{GPUTextureViewDescriptor/baseMipLevel}} &lt; {{GPUTextureDescriptor/mipLevelCount}}
+                - 1 &le; {{GPUTextureViewDescriptor/mipLevelCount}} &le; {{GPUTextureDescriptor/mipLevelCount}} - {{GPUTextureViewDescriptor/baseMipLevel}}
```

Comment:
[1] IMO clearer as

- view.baseMipLevel &ge; 0
- view.mipLevelCount &ge; 1
- view.baseMipLevel + view.mipLevelCount &le; texture.mipLevelCount

(and note the third rule makes "view.baseMipLevel &lt; texture.mipLevelCount" redundant)

We've done this in other places, with the assumption that numbers in the spec operate in the space of mathematical real numbers (or integers) and have no overflow behaviors.